src/lib/i18n/messages/en.json

```
@@ -285,6 +287,7 @@
 				"expand_all": "Expand All"
 			},
 			"moderator_only": "Moderator only",
+			"main_language": "The main language of the product",
```

Biome (2.5.1)

[error] The key main_language was already declared.

(lint/suspicious/noDuplicateObjectKeys)